src/routes/redirect/redirect.go

```diff
@@ -36,7 +36,11 @@ func redirectShortCode(ctx *fiber.Ctx) error {
 	if urlErr != nil {
 		var e *controllers.UrlError
 		if errors.As(urlErr, &e) {
-			return ctx.Status(fiber.StatusNotFound).JSON(dtos.CreateErrorResponse(e.ErrorDetails()))
+			if errors.Is(e, controllers.SpecificShortUrlLengthError) {
+				return ctx.Status(fiber.StatusBadRequest).JSON(dtos.CreateErrorResponse(e.ErrorDetails()))
```

Comment:
actually in the case of `GET`ing, we can just throw a 404 if short url > 10 chars because we defintely "cannot find it" in our db, as it only ha short urls < 10 chars. 

it is a bad request, when creating. but when fetching, it is just a not-found scenario.